src/app/community/community.module.ts

```diff
@@ -0,0 +1,27 @@
+import { RouterModule, Routes } from '@angular/router';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { BrowserModule } from '@angular/platform-browser';
+import { AngularFireModule } from 'angularfire2';
+import { NgModule } from '@angular/core';
+import { CommunityComponent } from './community.component';
+
+const appRoutes: Routes = [
+	{ path: 'community', component: CommunityComponent }
+];
+@NgModule({
+	declarations: [
+		CommunityComponent
+	],
+	imports: [
+		BrowserModule,
+		FormsModule,
+		HttpModule,
+		RouterModule.forRoot(appRoutes),
```

Comment:
I like doing this in a separate file and naming it like CommunityRoutingModule
See Here: https://angular.io/docs/ts/latest/tutorial/toh-pt5.html#!#refactor-routes-to-a-_routing-module_

Also these will be child routes so use
```typescript
RouterModule.forChild(routes)
```